x-pack/functionbeat/function/provider/cli.go

```diff
@@ -20,7 +21,13 @@ type CLIManager interface {
 
 	// Remove takes a function name and remove the specific function from the remote provider.
 	Remove(string) error
+
+	// Export prints exported function template to stdout.
+	Export(string) error
 }
 
+// ZipResourcesFunc is a function which returns the list of resources for each function type for a provider.
+type ZipResourcesFunc func() map[string][]bundle.Resource
```

Comment:
Looks like this is only used in the package command. Why not add 'Package' to the CLIManager to be consistent with the other commands?